DevCycle.SDK.Server.Common/Model/Variable.cs

```diff
@@ -177,5 +230,11 @@ private static bool IsNumericType(Type type)
                 default: return false;
             }
         }
+
+        public ResolutionDetails<T> GetResolutionDetails()
+        {
+            return new ResolutionDetails<T>(Key, Value, ErrorType.None,
+                IsDefaulted ? Reason.Default : Reason.TargetingMatch);
```

Comment:
Just to confirm - `isDefaulted` is only true if the value was returned based on no rule matches?

Reply:
No rule matches; or a variable type mismatch as part of the evaluation scheme - we're basing these off the existing implementation we have for JS - https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/openfeature-nodejs-provider/src/DevCycleProvider.ts